frontend/src/lib/dashboard/prepareQueryRangePayload.ts

```diff
@@ -91,8 +93,8 @@ export const prepareQueryRangePayload = ({
 	});
 
 	const queryPayload: QueryRangePayload = {
-		start: parseInt(start, 10) * 1e3,
-		end: parseInt(end, 10) * 1e3,
+		start: startTime ? startTime * 1e3 : parseInt(start, 10) * 1e3,
+		end: endTime ? endTime * 1e3 : parseInt(end, 10) * 1e3,
```

Comment:
can you help me understand, why do we need this?

Reply:
Sure @ahmadshaheer. Let's assume someone clicked on some log line which is the time as 2:30 PM then the start time will be 11:30 AM and the end time will be 5:30 PM.  We will subtract 3 hours for start time and add 3 hours for end time based on log line time.